ConsoleFramework/Controls/PasswordBox.cs

```diff
@@ -0,0 +1,278 @@
+﻿using System;
+using System.Collections.Generic;
+using ConsoleFramework.Core;
+using ConsoleFramework.Events;
+using ConsoleFramework.Native;
+using ConsoleFramework.Rendering;
+
+namespace ConsoleFramework.Controls
+{
+    public class PasswordBox : TextBox
+    {
+
+        private string mask;
```

Comment:
I think may be there is no need to store the mask at all ? If we store the text, the mask is just PassChar copied text.Length times. What do you think about it ?

Reply:
good idea might rewrite the class since this code after a more carefull review showed that it's full of bad practices from my old days.